.packit.yaml

```diff
@@ -29,5 +29,14 @@ jobs:
   metadata:
     targets:
     - fedora-29-x86_64
+    - fedora-30-x86_64
+    - fedora-31-x86_64
     - fedora-rawhide-x86_64
+- job: tests
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-29-x86_64
```

Comment:
https://www.linuxcompatible.org/story/fedora-29-eol-on-20191126/

Reply:
So we have another 6 days..;) I'll remove it.